new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/ActionConstants.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes.actions;
+
+public interface ActionConstants {
+
+  // Work directory for the integration test suite
+  public static final String WORK_DIR 
+      = System.getProperty("java.io.tmpdir") + "/it-results";
+  // Directory for resources
+  public static final String RESOURCE_DIR 
+      = System.getProperty("user.dir") + "/src/test/resources"; 
+  // Directory for all WDT models
+  public static final String MODEL_DIR = RESOURCE_DIR + "/wdt-models"; 
+  // Directory for download items
+  public static final String DOWNLOAD_DIR = WORK_DIR + "/download";
+  
+  // ------------ WebLogicImageTool action constants -------------
+  public static final String WLS = "WLS";
+  public static final String JRF = "JRF";
+  public static final String RJRF = "RestrictedJRF";
+  public static final String WLS_BASE_IMAGE_NAME 
+      = "container-registry.oracle.com/middleware/weblogic";
+  public static final String JRF_BASE_IMAGE_NAME 
+      = "container-registry.oracle.com/middleware/fmw-infrastructure";
+  public static final String WLS_BASE_IMAGE_TAG = "12.2.1.4";
+
+  public static final String DEFAULT_MODEL_IMAGE_NAME = "test-mii-image";
+  public static final String DEFAULT_MODEL_IMAGE_TAG  = "v1";
+  
+  // ------------ WebLogic Image Tool constants----------------------------
+  public static final String WIT = "WIT";
+  public static final String WDT = "WDT";
+
+  public static final String WIT_DOWNLOAD_URL 
+      = "https://github.com/oracle/weblogic-image-tool";
+  public static final String WIT_VERSION    = "release-1.8.3";
```

Comment:
I am okay to leave this here for now, but I think it would be easier to manage these versions from a property in the maven pom.xml.  Like <properties> <wit-version>release-1.8.3</wit-version> </properties> in the pom.xml for this test module.

Reply:
changed the code to get the version from pom.xml.